code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -137,6 +137,8 @@
 	. += span_notice("With the panel open:")
 	. += span_notice(" -Use a wrench with left-click to rotate [src] and right-click to unanchor it.")
 	. += span_notice(" -Use a multitool with left-click to change the piping layer and right-click to change the piping color.")
+	. += span_notice(" -[EXAMINE_HINT("AltClick")] to cycle between temperaure ranges.")
+	. += span_notice(" -[EXAMINE_HINT("CtrlClick")] to toggle on/off.")
```

Comment:
Could/Should these be screentips displayed on mouseover instead of/as well as examine hints?

Reply:
But screen tips are only used when the player holds something in their hand and point at the machine to explainwhat that item would do on that machine.

Displaying tips for an empty hand defeats that purpose??? I don't know it would be a very long screen tip and would look ugly mostly